source/dealer.js

```diff
@@ -74,113 +76,122 @@ class Dealer {
-        // loop over players and prompt for bets each players input updates hand.bet value and decriments player.bank
-        //the dealer doesn't bet and is the last person in the round, so current players are from 0 to length - 2. 
-        if (this.currentPlayer < this.round.length - 2) {
-          this.currentPlayer++;
-        }
-        else {
-          this.currentState = 'deal';
-          this.currentPlayer = 0;
-        }
-        break;
-      case 'deal':
-        //deals cards to everyone
-        this.deal();
-        this.currentState = 'player';
-        // if dealer hand === 21 Dealer.status == blackjack
-        break;
-      case 'player':
-        // TODO :insert logics that
-        // deals with one player at a time. prompting with actions available based on hand
-        // if hit run hit, if stand run stand, if handcount over 21 bust if handcount === 21 status= blackjack
-        //console.log(this.currentPlayer);
-        //once this player busts or stands 
-        if (this.currentPlayer === this.round.length - 1)
-          this.currentState = 'dealer';
-        else
-          this.currentPlayer++;
-
-        //do blah
-        break;
-      case 'dealer':
-        //do
-        // if dealer hand count >= 17 perform hit action
-        // if dealer hand = 17 && > 21 stand
-        // if dealer hand > 21 status.bust
-        this.currentState = 'payout';
-        break;
-      case 'payout':
-        var dealerCount = this.round[this.round.length - 1].count; //get the count of the dealer's hand first. 
-        if (dealerCount > 21) {
-          //the dealer busted. pay out all players who did not bust. 
-          this.round.forEach(hand => {
-            if (hand.player !== 'dealer') {
-              if (hand.count < 21) {
-                //payout 1x for people who got 20 or less points
-                hand.player.bank += hand.bet * 2;
-              } else if (hand.count === 21) {
-                //payout 1.5x for people who got 21
-                hand.player.bank += hand.bet * 2.5;
-              }
+      }
+      break;
+    case 'deal':
+      //deals cards to everyone
+      this.deal();
+      this.currentState = 'player';
+      // if dealer hand === 21 Dealer.status == blackjack
+      break;
+    case 'player':
+      // TODO :insert logics that
+      // deals with one player at a time. prompting with actions available based on hand
+      // if hit run hit, if stand run stand, if handcount over 21 bust if handcount === 21 status= blackjack
+      //console.log(this.currentPlayer);
+      //once this player busts or stands 
+      if (this.currentPlayer === this.round.length - 1)
+        this.currentState = 'dealer';
+      else
+        this.currentPlayer++;
+
+      //do blah
+      break;
+
+    case 'dealer':
+      var houseCount = this.round[this.round.length - 1].count;
+      
+      if (houseCount > 17){
+        this.hit(dealer.userID);
```

Comment:
hmmm... not sure what dealer.userID is here, but i'm thinking we just call this.hit with the string 'dealer' since we've been using that to determine the dealer's role anyway.